keyboards/dztech/dz60rgb/config.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x1219
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    DOU
+#define PRODUCT         DZ60RGB
+#define DESCRIPTION     DZ60 ARM RGB keyboard
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 14
+
+#define MATRIX_ROW_PINS { B1, B10, B11, B14, B12 }
+#define MATRIX_COL_PINS {A6, A7, B0, B13, B15, A8, A15, B3, B4, B5, B8, B9, C13, C14 }
+
+/* #define UNUSED_PINS {A0,A1,A2, A3,A4,A9,A10,C15,A13,A14,B2}*/
+
+#define DIODE_DIRECTION COL2ROW
+
+#define DEBOUNCE 0
+
+#define IS_COMMAND() ( \
```

Comment:
Since this is the default value, it should be removed. 

It gets defined automatically, if it's not already defined, so it's not needed here.